### ISSUE TITLE
Share track name from track history to another app

### ISSUE THREAD

**Issue by RAPXT** [segler-alex/RadioDroid#961]
Hi,

I would like to see the ability to share a track name to another app (e. g. YouTube, Spotify) from the track history.

Thank you!

**Comment by werman** [comment]
Hi, you can click on a track and then "Copy Info" and paste it wherever you like.

**Comment by RAPXT** [comment]
Yep, I know about that. But simply sharing track info to another app would be more convenient and save some steps, wouldn't you agree?
